wiki/compatibilty.md

```diff
@@ -0,0 +1,21 @@
+<div id="main_content_wrap" class="outer">
```

Comment:
I noticed the name of the file is misspelled.

Reply:
modified the file name